### ISSUE TITLE
Five failures surfaced by compiling 10 dark aprender-serve test targets (2 diagnosed, 3 open)

### ISSUE THREAD

**Issue by noahgift** [paiml/aprender#2482]
#2479 makes 10 `aprender-serve` test targets compile for the first time in months. 746 tests then ran; **741 pass, 5 fail.** These failures are not introduced by that PR — they are the first observation of state that has been true and invisible. Filing rather than papering over, and the targets are deliberately NOT added to `ci.yml:317` yet so main stays green.

Two of the five are already diagnosed, and neither is what its message suggests.

### `ffn_coverage::test_ffn_paths_differ` — degenerate fixture, not a GELU/SwiGLU defect

The message reads alarmingly: *"GELU and SwiGLU paths should produce different logits (diff=0)"*. Two activation paths producing byte-identical output would mean the gate weight is ignored.

It isn't. Probing the actual values, both paths return 100 logits that are **all exactly `-28.792973`** — identical *and* uniform across the whole vocab:

```
PROBE gelu[0..4]  =[-28.792973, -28.792973, -28.792973, -28.792973] sum=-2879.297 nonzero=100
PROBE swiglu[0..4]=[-28.792973, -28.792973, -28.792973, -28.792973] sum=-2879.297 nonzero=100
```

The fixtures do differ correctly (`ffn_gate_weight: None` vs `Some(..)`). The problem is `create_q4_0_data`, which fills every weight row from the same `(row + block + i) % 16` pattern. Every row is effectively identical, so the hidden state is constant and every vocab logit collapses to one value. **The test cannot distinguish the two paths no matter what the product does** — it would have failed on day one had it ever been compiled.

Fix: give the fixture row-varying weights, then re-assert. Until then this test proves nothing in either direction, so do not read its failure as an FFN defect.

### `api_coverage::test_completion_request_with_all_params` — f32/f64 precision

```
left: Some(0.699999988079071)   right: Some(0.7)
```
`0.7f32` widened, compared against an `f64` literal. Test artifact. Compare as `f32`, or use an epsilon.

## Needs investigation

| Test | Symptom | First read |
|---|---|---|
| `api_coverage::test_chat_completion_request_n_multiple` | `json.contains(r#""n":5"#)` fails | The `n` field (number of completions) is not serialized. Either a `skip_serializing_if` that should not apply, or `n` was dropped from the request type. **If `n` is silently ignored rather than rejected, that is a real API defect** — a client asking for 5 completions gets 1 with no error. |
| `driver_cpu::test_driver_cpu_forward_phi2_single_token` | `forward() with LayerNorm should succeed` | The LayerNorm (non-RMSNorm) path fails outright on a Phi-2-shaped config. Phi-2 is a supported architecture. |
| `gguf_config_coverage::test_cov_scratch_buffer_from_config` | `left: 3072, right: 6144` | Scratch buffer sized at half the expected. Could be a deliberate optimization that never updated the test, or an under-allocation. Worth pinning down which — an under-sized scratch buffer is the kind of thing that surfaces as corruption under load, not as a clean failure. |

## Why these were invisible

`workspace-test` runs `--lib`; `ci.yml:317` names ~19 integration targets out of **648** in the workspace. These 10 were not among them and did not compile either, so nothing ever reported them. The struct fields they were missing (`query_pre_attn_scalar`, `post_attn_norm_weight`, `post_ffw_norm_weight`, and the `ModelMetadataResponse` Option migration) had simply moved on without them.

Sequencing: fix the five, *then* add the targets to CI. Adding them first turns main red for a week.

Refs #2479, #2481, #2474.

**Comment by noahgift** [comment]
PP-066 triage 2026-09-05: **in 0.66 scope, no row yet** — row `G-5` proposed on #2873 (PR #2479 made 10 aprender-serve test targets compile; 5 failed and were 'deliberately NOT added to ci.yml' — they are still absent from the named-target list at ci.yml l.408 (ffn_coverage/gguf_coverage not listed), so the 3 undiagnosed failures stay invisible. The class row G-5 (every target compiles somewhere; enumeration derived from cargo metadata) is the fix.)

Instance of class #2878 — universe authored, not derived — targets, scopes and lists enumerated by hand so what is not on the list is never built, linted or run.

_PP-066 triage 2026-09-05 (issue #2873); ledger `docs/audits/pp-066-triage-2026-09-05.md`._

**Comment by noahgift** [comment]
triage 2026-09-23 (aprender-dd, standing triage owner): milestoned to 0.70.0, theme Fast Train (CI, gates, pv, release tooling, hygiene); it moved out of `backlog` (the 2026-09-17 overflow bucket, not a release).
